1.1/funcsrules.ttl

```diff
@@ -1256,7 +1257,7 @@ rules:sfCrosses a skos:Concept ;
     skos:definition "A query rewrite rule used to determine if two spatial objects cross based their associated primary geometry objects."@en ;
     skos:prefLabel "crosses"@en .
 
```

Comment:
Shouldn't sfDisjoint and sfTouches also be declared sd:Function?

Reply:
I think the functions are, but you pointed out the RIF rule definitions. 
As far as I know there is no equivalent to a RIF rule in the Service Description vocabulary